src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -59,7 +59,7 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
 
-                Ssl.SetProtocolOptions(innerContext, protocols);
+                Ssl.SetProtocolOptions(innerContext.DangerousGetHandle(), protocols);
```

Comment:
I should have expanded the lines above this change. :)  Ok, it's pretty obvious from the code that this is safe - but it's not obvious why we aren't just passing the SafeHandle here.